pwnagotchi/ui/web/templates/index.html

```diff
@@ -1,28 +1,30 @@
 <html>
 <head>
-    <title>{{ title }}</title>
-    <link rel="stylesheet" type="text/css" href="/css/style.css"/>
+  <title>{{ title }}</title>
+  <link rel="stylesheet" type="text/css" href="/css/style.css"/>
 </head>
 <body>
-<div class="full pixelated">
-    <img src="/ui" id="ui"/>
-    <br/>
-    <hr/>
-
+<div class="content">
+  <div class="image-wrapper">
+    <img class="ui-image pixelated" src="/ui" id="ui"/>
+  </div>
+  <div class="buttons-wrapper">
     <form class="action" method="POST" action="/shutdown"
           onsubmit="return confirm('This will halt the unit, continue?');">
-        <input style="display:inline;" type="submit" class="block" value="Shutdown"/>
-        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+      <input type="submit" class="button" value="Shutdown"/>
+      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
     </form>
 
     <form class="action" method="POST" action="/restart"
           onsubmit="return confirm('This will restart the service in {{ other_mode }} mode, continue?');">
-        <input style="display:inline;" type="submit" class="block" value="Restart in {{ other_mode }} mode"/>
-        <input type="hidden" name="mode" value="{{ other_mode }}"/>
-        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+      <input type="submit" class="button" value="Restart in {{ other_mode }} mode"/>
+      <input type="hidden" name="mode" value="{{ other_mode }}"/>
+      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
     </form>
+  </div>
 </div>
 
+<script type="text/javascript" src="/js/refresh.js"></script>
```

Comment:
i think this should be viewportHeight.js

Reply:
yep, changed :)